packages/gatsby-theme-docs/src/modules/ai-assistant/components/chat-modal.jsx

```diff
@@ -529,6 +598,105 @@ const ChatModal = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [currentChatMode]);
 
+  if (!isAuthenticated) {
+    return (
+      <FormDialog
+        size="scale"
+        title=""
+        labelPrimary="Send"
+        isOpen={!!aiAssistantModal}
+        onClose={() => {
+          closeAiAssistantModal();
+          setConversationId(createNewConversationId());
+        }}
+        displayPrimaryButton={!!chatConfig?.readOnly}
+        isPrimaryButtonDisabled={
+          !(formik.isValid && formik.dirty) || formik.isSubmitting
+        }
+        displaySecondaryButton={false}
+        onPrimaryButtonClick={() =>
+          !chatConfig?.readOnly && formik.submitForm()
+        }
+        background={designSystem.colors.light.surfaceSecondaryTopMenu}
+      >
+        <ChatContainer data-testid="ai-assistant-modal">
+          <LeftBlank />
+          <ChatSideArea>
+            <SideTopContainer>
+              <BackgroundWrapper>
+                <img src={robotPng} width={400} alt="robot icon" />
+              </BackgroundWrapper>
+            </SideTopContainer>
+            <ChatBottomContainer>
+              <DisclaimerText>{LEGAL_DISCLAIMER}</DisclaimerText>
+            </ChatBottomContainer>
+          </ChatSideArea>
+          <ChatMainArea>
+            <ChatMainAreaWhenLoggedOut>
+              <LoginButton
+                theme="primary"
+                label="Log in to start the Assistant"
+              />
+            </ChatMainAreaWhenLoggedOut>
+          </ChatMainArea>
+          <RightBlank />
+        </ChatContainer>
+      </FormDialog>
+    );
+  }
+
+  if (isNotValidatedUser(user)) {
+    return (
+      <FormDialog
```

Comment:
Do we really need to duplicate the whole `<FormDialog>` for not authenticated and not validated users? Isn't possible to use some conditionals within the existing `<FormDialog>` ?

Reply:
We could also do conditionals here but the code would be much harder to read in my opinion. However, we can move the two cases into separate components to reduce the code in the chat-modal component.

Reply:
--> https://github.com/commercetools/commercetools-docs-kit/pull/1892/commits/7451e4e2e94ca502b757e0d89b1cd4fa6379cba5